AIPs/AIP-11.md

```diff
@@ -0,0 +1,51 @@
+---
+AIP : AIP-11
+Title : 'Pikkle service', receiving Dapp Incubation Fund from AERGO
+Author : Allan KIM, Co-CEO of Blocko
+Status : Stage 1
+Category : dodona
+Created : 2021-04-23
+---
+
+## Pikkle Service Introduction
+Pikkle is a blcokchain-based voting and attendance check service, participated by KBA(Korean Broadcasters Association), and developed and operated by the Blocko, Aergo’s technical partner. Users will enjoy a new user experience that anyone can check information on voting and admission in real time even for the layperson of blockchain technology.
```

Comment:
Blcokchain --> Blockchain

Reply:
@hanlsin Thank you for your feedback, I'll modify typo.